### PR TITLE
HTML5 Client sound - added sound for unreadMessages

### PR DESCRIPTION
**Pull request by JaeeunCho**
added sound for unreadMessages

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx [hunk 1]
```diff
@@ -88,6 +93,7 @@ class NavBar extends Component {
         <div className={styles.right}>
           <SettingsDropdown />
         </div>
+        {hasUnreadMessages ? this.soundForUnreadeMessages() : null}
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx]: I dont think having this on the `render` method of the navBar makes sense. Proably should be moved to the App component

File: bigbluebutton-html5/imports/ui/components/app/component.jsx [hunk 2]
```diff
@@ -149,7 +150,17 @@ export default class App extends Component {
     return false;
   }
 
+  renderSoundUnreadeMessages() {
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/app/component.jsx]: Theres nothing being render in this function therefore makes no sense to render it.

Probably you should check in the `componentDidUpdate` if you `hadUnreadMessages` and then perform the play action.

File: bigbluebutton-html5/imports/ui/components/app/container.jsx [hunk 3]
```diff
@@ -61,6 +68,9 @@ export default createContainer(() => {
     wasKicked: wasUserKicked(),
     isLoading: getLoading(),
     modal: getModal(),
+    unreadMessageCount: checkUnreadMessages(),
+    openChats: userListService.getOpenChats(params.chatID).map(chat => chat.id),
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/app/container.jsx]: I would recommend moving the logic to a helper function in the same file

File: bigbluebutton-html5/imports/ui/components/app/container.jsx [hunk 4]
```diff
@@ -13,6 +13,8 @@ import NavBarContainer from '../nav-bar/container';
 import ActionsBarContainer from '../actions-bar/container';
 import MediaContainer from '../media/container';
 import ClosedCaptionsContainer from '../closed-captions/container';
+import userListService from '../user-list/service';
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/app/container.jsx]: Should be PascalCase. `UserListService`

File: bigbluebutton-html5/imports/ui/components/app/container.jsx [hunk 5]
```diff
@@ -51,16 +53,28 @@ const setLoading = (val) => {
...
+                        .filter(userID => userID !== Auth.userID);
+};
+
+export default createContainer(({ params }) => {
   Promise.all(subscribeForData())
   .then(() => {
     setLoading(false);
   });
 
+  const openChats = () => {
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/app/container.jsx]: Should be moved to outside the `createContainer` callback.

Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/app/container.jsx]: this was initially written as return {.... openChats: userListService.getOpenChats(params.chatID).map(chat => chat.id), .......}
Would keeping it that way be an option?

**Pull request merged**